### PR TITLE
Suggestions for improving the docs

### PR DESCRIPTION
**Pull request by landerlini**
# Summary

I collected in this PR some possible fixes to the docs.

The only non-editorial correction is replacing here and there the word "sidecar" with "plugin" or "container plugin".
In most implementations of interlink plugin, the plugin is not a sidecar container as it is intended in Kubernetes and this is generating confusion.

### CODE REVIEW

File: .idea/interLink.iml
Comment (dciangot) [inline .idea/interLink.iml:1]: @landerlini is this neeeded useful by the IDE? can be removed or do you think that it might be useful?

**Review by dciangot — COMMENTED** [review]
LGTM, only one minor comment on the IDE specific file committed

File: .idea/interLink.iml
Comment (landerlini) [reply .idea/interLink.iml:1]: Ciao Diego, 
  can be dropped, I am not sure why PyCharm added these lines.

Lucio

**Review by dciangot — APPROVED** [review]
LGTM

**Pull request merged**